test/release/examples/users-guide/base/configs.execopts

```diff
@@ -0,0 +1,4 @@
+./a.out --n 1000
+./a.out --n=1000
+./a.out --epsilon=0.000001 --debug=true  # configs-eps+debug.good
+./a.out --epsilon=0.000001 --debug       # configs-eps+debug.good
```

Comment:
All that's needed is for literalinclude to strip the # comments when reading .compopts/.execopts. Right?